app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
```

Comment:
```suggestion
  def home; end
```

Reply:
Do i repeat this for help and about too?